.github/workflows/ci-build.yml

```diff
@@ -39,7 +39,7 @@ jobs:
       # Make sure we use the latest version of MatrixKit
       - name: Reset MatrixKit pod
         run: rm -rf Pods/MatrixKit
-
+        
```

Comment:
I think there is a mistake here 
we should remove MatrixKit pod and keep MatrixSDK pod (like in element-ios [here](https://github.com/vector-im/element-ios/blob/develop/.github/workflows/ci-build.yml#L40)
Don't you think so?

Reply:
Changes here are just to clean the cache for CI.
The reset MatrixKit will be replaced by reset MatrixSDK with the next rebase.